Resources/views/CRUD/edit_orm_one_to_many_inline_table.html.twig

```diff
@@ -38,7 +38,7 @@ file that was distributed with this source code.
                     <td class="
                         sonata-ba-td-{{ id }}-{{ field_name  }}
                         control-group
-                        {% if nested_field.vars.errors|length > 0 %} error sonata-ba-field-error{% endif %}
+                        {% if nested_field.vars.errors is defined and nested_field.vars.errors|length > 0 %} error sonata-ba-field-error{% endif %}
```

Comment:
I think `if nested_field.vars.errors|default([])|length > 0` would be better.

Reply:
It's not a problem in performance point of view? default will create an empty array for nothing, and this code run into a loop.

Reply:
I think you won't have any issue finding code that dwarves this potential problem.